src/test/java/org/alliancegenome/curation_api/bulkupload/MoleculeBulkUploadITCase.java

```diff
@@ -0,0 +1,387 @@
+package org.alliancegenome.curation_api.bulkupload;
+
+import java.io.IOException;
+import java.nio.file.*;
+
+import org.alliancegenome.curation_api.resources.TestElasticSearchResource;
+import org.junit.jupiter.api.*;
+
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.junit.QuarkusIntegrationTest;
+import io.restassured.RestAssured;
+import io.restassured.config.*;
+
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.nullValue;
+
+@QuarkusIntegrationTest
+@QuarkusTestResource(TestElasticSearchResource.Initializer.class)
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class MoleculeBulkUploadITCase {
+
+    @BeforeEach
+    public void init() {
+        RestAssured.config = RestAssuredConfig.config()
+                .httpClient(HttpClientConfig.httpClientConfig()
+                    .setParam("http.socket.timeout", 100000)
+                    .setParam("http.connection.timeout", 100000));
+    }
+
+    @Test
+    @Order(1)
+    public void moleculeBulkUploadCheckModExamples() throws Exception {
+        String content = Files.readString(Path.of("src/test/resources/bulk/05_molecule/00_mod_examples.json"));
+
+        // upload file
+        RestAssured.given().
+            contentType("application/json").
+            body(content).
+            when().
+            post("/api/molecule/bulk/moleculefile?async=false").
```

Comment:
I believe we removed the async param in the API so the `?async=false` could be removed from all these URL's?